modules/api/README.md

```diff
@@ -0,0 +1,38 @@
+# API
+
+## Purpose
+
+The api module is intended to provide a stable, versioned REST interface
+for interacting with the core LORIS functionality without requiring backend
+SQL access.
+
+## Intended Users
+
+The module has 2 intended users:
+1. Users (or creators) of LORIS data that are external to the LORIS core
+   software that must extract or import data from LORIS.
+2. Internal LORIS modules
+
+## Scope
+
+The API only provides information about concepts that are cross-module
+or data that is of general concern to users. 
+
+NOT in scope:
+
+- Data and functionality that is specific to a single module, and not of general
+  nature.
+- A graphical frontend
+
+## Permissions
+
+????
```

Comment:
I could use some help with this part from anyone who knows the current state of it.

Reply:
@driusan maybe it should be stated that a login session needs to be  retrieved before accessing other API requests? @xlecours

Reply:
I was thinking I needed help with permissions which affect what data can be accessed, but that's a good point, I'll add that too.